lcls_tools/common/frontend/plotting/image.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+from matplotlib import pyplot as plt, patches
+
+
+def plot_image_projection_fit(fit_object, results, n_stds=3):
+    """
+    plot image and projection data for validation
+    """
+    fig, ax = plt.subplots(3, 1)
+    fig.set_size_inches(4, 9)
+
+    image = results["processed_image"]
```

Comment:
part of the goal of this function is to easily validate that the image analysis proceeded successfully and to debug any issues. Given the stochastic nature of the fitting process it would be useful to examine the results of fitting in a convenient way to debug / verify proper fitting